dropbox-sdk-dotnet/Dropbox.Api.Tests/DropboxApiTests.cs

```diff
@@ -91,6 +108,130 @@ public void Cleanup()
             }
         }
 
+        /// <summary>
+        /// Tests creating a client with only refresh token and
+        /// ensuring the client refreshed the token before making a call
+        /// </summary>
+        /// <returns>The <see cref="Task" /></returns>
+        [TestMethod]
+        public async Task TestRefreshClient()
+        {
+            var client = new DropboxClient(UserRefreshToken, AppKey, AppSecret);
+            var result = await client.Users.GetCurrentAccountAsync();
+            Assert.IsNotNull(result.Email);
+        }
+
+        /// <summary>
+        /// Test get authorization url
+        /// </summary>
+        /// <returns>The <see cref="Task"/></returns>
+        [TestMethod]
+        public async Task TestGetAuthorizationUri()
+        {
+            string clientId = "myclientid";
+            string[] redirectUris = new[] {"", "http://127.0.0.1:52475/"};
+            string[] states = new[] {"", "state"};
+            bool[] forceReapproves = new[] {false, true};
+            bool[] disableSignups = new[] {false, true};
+            string[] requireRoles = new[] {"", "role"};
+            bool[] forceReauthentications = new[] {false, true};
+            TokenAccessType[] tokenAccessTypes = new[]
+                {TokenAccessType.Legacy, TokenAccessType.Offline, TokenAccessType.Online};
+            foreach (string redirectUri in redirectUris)
+            {
+                foreach (var state in states)
+                {
+                    foreach (var forceReapprove in forceReapproves)
+                    {
+                        foreach (var disableSignup in disableSignups)
+                        {
+                            foreach (var requireRole in requireRoles)
+                            {
+                                foreach (var forceReauthentication in forceReauthentications)
+                                {
+                                    foreach (var tokenAccessType in tokenAccessTypes)
+                                    {
+                                        var authUri = DropboxOAuth2Helper.GetAuthorizeUri(OAuthResponseType.Code,
```

Comment:
Looks like this only tests auth URI generation and not the actual requesting of initial SLT and refresh flow. Will they be added when PKCE and scopes are added?